google/datalab/ml/_feature_slice_view.py

```diff
@@ -58,10 +58,8 @@ def plot(self, data):
     """
     import IPython
 
-    if sys.version_info.major > 2:
-      basestring = (str, bytes)  # for python 3 compatibility
-
-    if isinstance(data, basestring):
+    if ((sys.version_info.major > 2 and isinstance(data, str)) or 
+       (sys.version_info.major <= 2 and isinstance(data, basestring))):
```

Comment:
it might be cleaner to use python's six package and do

if isinstance(data, six.string_types)